uwblueprint/planet-read
# Set up languages table

## Notion ticket link



[Set up languages table](https://www.notion.so/uwblueprintexecs/Set-up-languages-table-98204fe56cee4b3892c061b6dd153494)



## Implementation description

- create migration for languages table
- insert languages from `LanguageEnum` by default



## Steps to test

1. Run the migration: `docker exec -it planet-read_py-backend_1 /bin/bash -c "flask db upgrade"`
2. Check that languages table exists: `docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; SELECT * FROM languages;"`
3. Check that there are 4 rtl languages by default: `docker exec -it planet-read_db_1 mysql -u root -proot -e "USE planet-read; SELECT * FROM languages where is_rtl=True;"` (Arabic, Hebrew, Kurdish, Urdu)
4. Check that downgrading works properly: `docker exec -it planet-read_py-backend_1 /bin/bash -c "flask db downgrade"`



## What should reviewers focus on?

- do we want to insert languages by default?

## Checklist
